packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -287,13 +291,22 @@ class IOSDevice extends Device {
         await diagnoseXcodeBuildFailure(buildResult);
         printError('');
         return LaunchResult.failed();
+      }else{
```

Comment:
Not sure why the analyzer isn't mad about this, but there's a space missing between the `}` and `else`